devtools/test_dashboard/devtools.js

```diff
@@ -8,6 +8,8 @@ var credentials = require('../../build/credentials.json');
 var Lib = require('@src/lib');
 var d3 = Plotly.d3;
 
+require('./perf');
```

Comment:
cc @rreusser would be nice to have something like that in https://github.com/rreusser/plotly-mock-viewer  

Moreover, @dfcreative's brilliant :sparkles: https://www.npmjs.com/package/fps-indicator :sparkles: would be a nice addition to our dev tools.